src/status_im/multiaccounts/recover/core.cljs

```diff
@@ -20,7 +20,7 @@
 
 (defn existing-account?
   [root-key multiaccounts]
-  (contains? multiaccounts (:key-uid root-key)))
+  (contains? multiaccounts (:keyUid root-key)))
```

Comment:
why do we have this key mess?

Reply:
i mean why don't we store and use in clojure only kebab ?

Reply:
@flexsurfer i din't check when and why it was changed, can't answer

Reply:
@flexsurfer it didn't work properly at least 10 weeks ago, bisection is a nightmare